apiserver/facades/schema.json

```diff
@@ -848,6 +848,267 @@
             }
         }
     },
+    {
+        "Name": "Admin",
+        "Description": "admin is the only object that unlogged-in clients can access. It holds any\nmethods that are needed to log in.",
```

Comment:
I don't believe this is true anymore, we should update the description.

Reply:
CMR callers can access various facades, but they do need a valid macaroon to get past auth